Wikipedia/UI-V5/TableOfContentsReadMoreItem.swift

```diff
@@ -13,7 +13,15 @@ public protocol TableOfContentsFooterItem : TableOfContentsItem {
 }
 
 public class TableOfContentsReadMoreItem : NSObject, TableOfContentsFooterItem {
-    public let titleText: String = localizedStringForKeyFallingBackOnEnglish("article-read-more-title")
+    let site:MWKSite
+    init(site: MWKSite) {
+        self.site = site
```

Comment:
yeah - anything that is a `let` needs to be set before you call init - since that is a constant (can't change later). for instance you can see the delegate ivar in the doc view controller is just a `var`. so it doesn't need to be set.